### ISSUE TITLE
Failed to Connect to Sql Server with MultiSubnetFailover .netCore 2.0

### ISSUE THREAD

**Issue by ezeasharma** [dotnet/corefx#23984]
I'm running a .NET Core App 2.0 in a container against a SqlServer 12 database with MultiSubnetFailover.

My connection string looks like this...

Data Source=***.***.***.com;Initial Catalog=DATABASENAME;User ID=USER_ID;Password="USER_PASSWORD";Connect Timeout=60;MultiSubnetFailover=True

When i try to connect, it fails with the following error

Message="A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections.

Same issue is documented here for 1.1
https://github.com/dotnet/corefx/issues/14978

**Comment by ezeasharma** [comment]
Any updates on this one? Seems like a major issue.

**Comment by saurabh500** [comment]
cc @corivera

**Comment by corivera** [comment]
@ezeasharma Were you able to resolve this issue?

**Comment by ezeasharma** [comment]
Nope. Still seeing the same behaviour. Is there any patch released on top of 2.0?

**Comment by corivera** [comment]
There's a preview build for 2.1 available at https://www.nuget.org/packages/System.Data.SqlClient/4.5.0-preview1-26216-02

Does explicitly specifying the port in the DataSource fix this?

**Comment by ezeasharma** [comment]
@rzachariah

**Comment by ezeasharma** [comment]
We will give both a shot but it could take few days. will report back here.

**Comment by epoyraz** [comment]
are there any news on this issue?
